trail/upload/process_uploads/standardizers/gmosn_standardizer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Class that facilitates header metadata translation for Gemini North instrument
+"""
+
+from datetime import datetime, timedelta
+
+from astropy.time import Time
+
+from upload.process_uploads.header_standardizer import HeaderStandardizer, StandardizeWcsException
+from upload.models import Metadata, Wcs
+
+
+__all__ = ["GmosnStandardizer", ]
+
+
+class GmosnStandardizer(HeaderStandardizer):
+
+    name = "gmosn_standardizer"
+    priority = 2
+
+    def __init__(self, header, **kwargs):
+        super().__init__(header, **kwargs)
+
+    @classmethod
+    def canStandardize(self, header, **kwargs):
+        lat = header.get("GEOLAT", False)
+        if lat and 19.8200 == lat: return True
+        return False
+
+    def standardizeMetadata(self):
+
+        # TODO: Fix datetimes
+        # There is a timesys key but I have no idea how to generically instantiate
+        # timezone aware datetime and astropy Time seems not to work well with
+        # Django - the astrometadata is also broken!
+        DATEOBS = self.header["DATE-OBS"]
+        EXP = self.header["EXPTIME"]
+        dec = DATEOBS.find('.')
+        if dec != -1: date = DATEOBS[:dec]
+        begin = datetime.fromisoformat(date)
+        end = begin + timedelta(seconds= EXP)
+
+        # TODO: filter out what is the filter standardization here?
+        # After uploading, no "astrometry.net key" runtime err?
+        # Do we need science program?
+        meta = Metadata(
+            obs_lon=self.header["GEOLON"],
+            obs_lat=self.header["GEOLAT"],
+            obs_height= 4213, #height in meters from official website
+            datetime_begin=begin,
+            datetime_end= end,
+            telescope="Gemini North",
+            instrument="Hamamatsu CCD", #Instrument given from official
+            exposure_duration=self.header["EXPTIME"],
+            filter_name=self.header["FILTER"].strip()
+        )
+        
+        return meta
+
+    def standardizeWcs(self, **kwargs):
+        # ignores any WCS information in the header and instead just sends it to astrometry.net to solve.
```

Comment:
Hmm. Can you elaborate on why do we need to force this to go to Astrometry.NET? 

I wanted the default logic flow to be 1) try to read the header for WCS --> if it fails: send to astrometry.net; else convert. This is how it's currently written [here](https://github.com/dirac-institute/trailblazer/blob/main/trail/upload/process_uploads/header_standardizer.py#L314). I see MOA has the same standardizer method, but are these really necessary or did you just cargo-cult it over?

If it's necessary then we should fix the code, but if it was cargo-culted then we can delete these methods because that's what's going to happen anyhow.